app/discovery/provider/docker.go

```diff
@@ -27,41 +29,29 @@ import (
 type Docker struct {
 	DockerClient DockerClient
 	Excludes     []string
-	Network      string
 	AutoAPI      bool
 }
 
 // DockerClient defines interface listing containers and subscribing to events
 type DockerClient interface {
-	ListContainers(opts dc.ListContainersOptions) ([]dc.APIContainers, error)
-	AddEventListenerWithOptions(options dc.EventsOptions, listener chan<- *dc.APIEvents) error
+	ListContainers() ([]containerInfo, error)
 }
 
-// containerInfo is simplified docker.APIEvents for containers only
+// containerInfo is simplified view of container metadata
```

Comment:
I'm not sure if I like such a coupling between docker cleint implementation and service-level containerInfo. Will be better to leave them separate and not to adopt with json lables for docker client request but rather fill it (copy) explicitly

Reply:
Just wanted to save some space and this somehow seemed like a good idea. I see it now. Fixed.